html/user/server_status.php

```diff
@@ -183,6 +185,14 @@ function show_status_html($x) {
         ";
     }
     end_table();
+
+    global $server_version;
+    global $git_commit;
```

Comment:
Those two `global` statements are not needed since you already declared both variables global in `release.inc`.